EngineeringModel/Views/Dialogs/ActualFiniteStateListDialog.xaml

```diff
@@ -13,6 +13,7 @@
              xmlns:cdp4Composition="clr-namespace:CDP4Composition;assembly=CDP4Composition"
              xmlns:dxe="http://schemas.devexpress.com/winfx/2008/xaml/editors"
              xmlns:engineeringModelData="clr-namespace:CDP4Common.EngineeringModelData;assembly=CDP4Common"
+             xmlns:xtraEditors="clr-namespace:DevExpress.XtraEditors;assembly=DevExpress.XtraEditors.v19.2"
```

Comment:
Is this extra reference necessary?

Reply:
Good point it has been added while I was trying to achieve something which is not required at last